taipy/core/_entity/_reload.py

```diff
@@ -89,7 +90,7 @@ def _do_set_entity(self, *args, **kwargs):
 
 
 @functools.lru_cache
-def _get_manager(manager: str):
+def _get_manager(manager: str) -> _Manager:
```

Comment:
I believe we should remove `_get_manager` method. It is overengineering to me.
It does not make the code easier to read or maintain.
It is just an indirection to me.

Reply:
Yes, in the _reload.py, it makes sense to keep it. I agree.